nav2_sms_recovery/src/sms_recovery.cpp

```diff
@@ -21,42 +21,45 @@ SMSRecovery::~SMSRecovery()
 
 void SMSRecovery::onConfigure()
 {
-  node_->declare_parameter("account_sid");
-  _account_sid = node_->get_parameter("account_sid").as_string();
-  node_->declare_parameter("auth_token");
-  _auth_token = node_->get_parameter("auth_token").as_string();
-  node_->declare_parameter("from_number");
-  _from_number = node_->get_parameter("from_number").as_string();
-  node_->declare_parameter("to_number");
-  _to_number = node_->get_parameter("to_number").as_string();
-  _twilio = std::make_shared<twilio::Twilio>(_account_sid, _auth_token);
+	auto node = node_.lock();
+
+	node->declare_parameter("account_sid");
+	_account_sid = node->get_parameter("account_sid").as_string();
+	node->declare_parameter("auth_token");
+	_auth_token = node->get_parameter("auth_token").as_string();
+	node->declare_parameter("from_number");
+	_from_number = node->get_parameter("from_number").as_string();
+	node->declare_parameter("to_number");
+	_to_number = node->get_parameter("to_number").as_string();
+	_twilio    = std::make_shared <twilio::Twilio>(_account_sid, _auth_token);
 }
 
-Status SMSRecovery::onRun(const std::shared_ptr<const Action::Goal> command)
+Status SMSRecovery::onRun(const std::shared_ptr <const Action::Goal> command)
 {
-  std::string response;  
-  bool message_success = _twilio->send_message(
-    _to_number, 
-    _from_number, 
-    command->message,
-    response,
-    "",
-    false);
-
-  if (!message_success) {
-    RCLCPP_INFO(node_->get_logger(), "SMS send failed.");
-    return Status::FAILED;
-  }
-
-  RCLCPP_INFO(node_->get_logger(), "SMS sent successfully!");
-  return Status::SUCCEEDED;
+	auto node = node_.lock();
+	std::string response;
+	bool message_success = _twilio->send_message(
+		_to_number,
+		_from_number,
+		command->message,
+		response,
+		"",
+		false);
+
+	if (!message_success)
```

Comment:
This is not properly linted, you removed the open bracket from the if line, which is the C++ google styling that we follow.

Reply:
can you please post the `uncrustify.cfg` file that you use as a gist ? 
I have hard time to find the exact settings that is used across navigation software

Reply:
I don't have one, use `ament_uncrustify` and it's preconfigured with the proper styling. Same with ament_cpplint.